yt_dlp/extractor/manyvids.py

```diff
@@ -1,113 +1,100 @@
-import re
+
 
 from .common import InfoExtractor
-from ..utils import (
-    determine_ext,
-    extract_attributes,
-    int_or_none,
-    str_to_int,
-    url_or_none,
-    urlencode_postdata,
-)
+from .. import traverse_obj
+from ..utils import determine_ext, int_or_none, parse_count, parse_duration, parse_iso8601, url_or_none
 
 
 class ManyVidsIE(InfoExtractor):
-    _WORKING = False
+    _WORKING = True
     _VALID_URL = r'(?i)https?://(?:www\.)?manyvids\.com/video/(?P<id>\d+)'
-    _TESTS = [{
-        # preview video
-        'url': 'https://www.manyvids.com/Video/133957/everthing-about-me/',
-        'md5': '03f11bb21c52dd12a05be21a5c7dcc97',
-        'info_dict': {
-            'id': '133957',
-            'ext': 'mp4',
-            'title': 'everthing about me (Preview)',
-            'uploader': 'ellyxxix',
-            'view_count': int,
-            'like_count': int,
+    _TESTS = [
+        {
```

Comment:
revert the formatting change here

the download test outputs a dict indented for the `TESTS = [{` style formatting